lib/src/models/fancy_dio_inspector/fancy_dio_inspector_tile_options.dart

```diff
@@ -13,8 +13,12 @@ class FancyDioInspectorTileOptions {
   /// then the string will be truncated and `...` will be appended at the end.
   final int? maxLines;
 
+  /// It controls whether to show the search field to filter elements or not.
+  final bool showSearch;
+
   const FancyDioInspectorTileOptions({
-    this.showButtons = true,
+    this.showButtons = false,
```

Comment:
`showButtons` has to be `true` as default.

Reply:
Good catch 👍